src/Models/TupleChanges.php

```
@@ -7,9 +7,9 @@
 use OpenFGA\Schema\CollectionSchemaInterface;
 
 /**
- * @extends AbstractIndexedCollection<TupleChange>
+ * @extends IndexedCollection<TupleChange>
  */
-final class TupleChanges extends AbstractIndexedCollection implements TupleChangesInterface
+final class TupleChanges extends IndexedCollection implements TupleChangesInterface
```

PHPStan (2.1.15)

Class OpenFGA\Models\IndexedCollection specifies template type T of interface OpenFGA\Models\IndexedCollectionInterface as OpenFGA\Models\TupleChange but it's already specified as OpenFGA\Models\TupleChangeInterface.


(generics.interfaceConflict)